src/hooks/useFetchUser.ts

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from '@tanstack/react-query';
+import { useWallet } from 'src/providers';
+import type { FeatureCardData } from 'src/types';
+
+export interface UseFetchUserProps {
+  featureCards: FeatureCardData[] | undefined;
+  isSuccess: boolean;
+  isConnected: boolean;
+}
+
+const STRAPI_CONTENT_TYPE = 'jumper-users';
+export const useFetchUser = (): UseFetchUserProps => {
```

Comment:
What do you think about changing the name to `usePersonalizedFeatureCards` or if we have any other information about the user we can rename it to `useUser`.

Reply:
I disagree on that one. I believe useUser is not very descriptive: We have an actual collection type for "Users". Whenever we will have to fetch users - I´d rather call that "useUsers". Here, it was just about fetching "personalized" feature cards by just adding one more filter.

Reply:
Okay, the first suggestion still applies 😄